source/modules/MEI.cmn.xml

```diff
@@ -876,17 +926,16 @@
     </attList>
   </classSpec>
   <classSpec ident="att.numberPlacement" module="MEI.cmn" type="atts">
-    <desc>Attributes that record the placement and visibility of numbers that accompany a bowed
-      tremolo or tuplet.</desc>
+    <desc>Attributes that record the placement and visibility of numbers that accompany a notational object.</desc>
     <attList>
       <attDef ident="num.place" usage="opt">
-        <desc>States where the tuplet number will be placed in relation to the note heads.</desc>
+        <desc>Indicates where the number is written in relation to its reference object.</desc>
```

Comment:
The data type, `data.POSITION.cmn.basic` says "`<desc>Location of symbol relative to a staff.</desc>`" but this says "Indicates where the number is written in relation to its reference object."

If the data type specifies that it's in relation to the staff, then I don't see how the instance of the data type can be more generic ('...in relation to its reference object...')

Reply:
Thanks for catching this, @ahankinson. Actually, I didn't pay much attention to the `<desc>`s yet. Will look for better descriptions…